packages/compartment-mapper/src/node-modules.js

```diff
@@ -469,6 +492,17 @@ const graphPackages = async (
       `Cannot find package.json for application at ${packageLocation}`,
     );
   }
+
+  // Resolve common dependencies.
+  const commonDependencyDescriptors = {};
+  for (const [alias, dependencyName] of Object.entries(commonDependencies)) {
+    const spec = packageDescriptor.dependencies[dependencyName];
```

Comment:
This may be sensitive in packages that don’t describe their `dependencies`, unless we’ve normalized the package descriptor above. Worth verifying.